ecommerce/extensions/payment/processors/payu.py

```diff
@@ -55,12 +55,12 @@ def __init__(self, site):
         """
         super(Payu, self).__init__(site)
         configuration = self.configuration
-        self.payment_page_url = configuration.get("payment_page_url")
-        self.merchant_id = configuration.get("merchant_id")
-        self.account_id = configuration.get("account_id")
-        self.api_key = configuration.get("api_key")
-        self.tax = configuration.get("tax")
-        self.tax_return_base = configuration.get("tax_return_base")
+        self.payment_page_url = configuration["payment_page_url"]
+        self.merchant_id = configuration["merchant_id"]
+        self.account_id = configuration["account_id"]
+        self.api_key = configuration["api_key"]
+        self.tax = configuration["tax"]
+        self.tax_return_base = configuration["tax_return_base"]
```

Comment:
is this totally safe?

Reply:
Actually, this was a slip from my part on the previous commits. It should raise a KeyError if some configuration is missing, as stated in the doc string.